pyproject.toml

```diff
@@ -58,6 +58,13 @@ flake8 = "^5.0.1"
 meson-python = "^0.14.0"
 cython = "^3.0.3"
 ninja = "^1.11.1.1"
+cibuildwheel = "^2.16.2"
+
+[tool.cibuildwheel]
+build = "cp39-* cp310-* cp311-* cp312-*"
+skip = ["pp*", "*i686", "*win32"]
```

Comment:
We have to skip these because of  https://github.com/numpy/numpy/issues/24703#issuecomment-1731269424
 and there were also problems with win32 builds.